android/app/src/main/AndroidManifest.xml

```diff
@@ -51,6 +51,9 @@
               <data
                 android:scheme="https"
                 android:host="tiktok.com" />
+              <data
+                android:scheme="https"
+                android:host="vm.tiktok.com" />
```

Comment:
This looks good. What links include the vm subdomain?

Reply:
Direct links to video.

Like https://vm.tiktok.com/ZSpdohha/ for example that is expanded to https://www.tiktok.com/@johnvanb/video/6870536778642771201 with some garbage at the end.

To create a vm link just share a video from TikTokLite.